web/website/content/faq.md

```diff
@@ -6,11 +6,15 @@ Here are some of the most common questions we hear. Have something else you'd
 like to ask? Pop by our [Discord](https://discord.com/invite/eQcfaCmsNc) and ask
 away!
 
-{{< faq "Cool story Hansel, but what can I actually do with PRQL now?" >}}
+{{< faq "Cool story Hansel, but what can I actually do with PRQL _now_?" >}}
 
-We're still early, and the opportunities for using PRQL are focused on two
-integrations:
+PRQL builds powerful SQL queries today. In addition, we are focusing on several
+integrations. Here's how you can use PRQL:
 
+- **Do real work with PRQL** Prototype your PRQL queries in the
```

Comment:
Maybe move this to the bottom and say something like "Prototype queries"? "Do real work" doesn't seem to be specific to this approach?

Reply:
I'm still casting about to find a way to describe that PRQL _is_ good enough to use right now. (And I think that talking about bindings to other languages makes its relevance seem like a distant possibility.

I'm content with what's here, and I'm running out of time in the near term to devote much more to this page (Tax time in the US (!)). I am completely content to let others continue to edit.

Reply:
> I'm content with what's here, and I'm running out of time in the near term to devote much more to this page (Tax time in the US (!)). I am completely content to let others continue to edit.

Totally empathize, it's important we can make small improvements without being dragged into a total refactor. 

I'll let this sit for a bit to see if others have thoughts; then happy to merge and I'll take a pass?

Reply:
I agree that we can emphasize that Playground can be used to assist in writing SQL immediately.

I actually use the SQL generated using Playground (after tweaking it), so it would be great if we could emphasize that it can be used right out of the box without installing anything.

(BTW, I look forward to the integration with DBeaver as I often execute SQL in DBeaver.)

Reply:
Perhaps a link to this might explain how PRQL actually works?
https://eitsupi.github.io/querying-with-prql/